CONTRIBUTING.md

```diff
@@ -308,39 +308,57 @@ You can find an installation guide for all the different environments here:
 ## Product Management
 
 The ZITADEL Team works with an agile product management methodology.
-You can find all the issues prioritized and ordered in the [product board](https://github.com/orgs/zitadel/projects/1/views/2).
+You can find all the issues prioritized and ordered in the [product board](https://github.com/orgs/zitadel/projects/2/views/1).
 
 Every two weeks the team goes through all the new issues and decided about the priority, effort and if it is ready to start or in the backlog.
```

Comment:
Could we add when the sprint is starting (Tuesdays?) and how that impacts release (next monday?). Would just be good to have somewhere, so we can link or look it up.

Reply:
Ah just found the part below (maybe mention: "see below")